components/form/FormList.tsx

```diff
@@ -26,7 +26,10 @@ const FormList: React.FC<FormListProps> = ({ children, ...props }) => {
   return (
     <List {...props}>
       {(fields, operation) => {
-        return children(fields.map(field => ({ ...field, fieldKey: field.key })), operation);
+        return children(
+          fields.map(field => ({ ...field, fieldKey: field.key })),
+          operation,
+        );
```

Comment:
git commit hook auto-formatted this line based on the .prettierrc printWidth setting, ideally I would add this to a new commit but didn't see until PR created